inc/Auth/SyncPermissions.php

```diff
@@ -47,7 +47,7 @@ public static function can_sync(
 		[ $entity_kind, $entity_name ] = $parts;
 
 		// Handle post type entities
```

Comment:
This comment should be updated to note that we now support collections, and that so collections are also being excluded here.